docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -697,8 +791,531 @@ To read and write a value of the Bluetooth descriptor:
+   };
+
+   var characteristicInitData = {
+     uuid: '9876',
+     descriptors: [descriptorInitData]
+     /* Define more attributes, if needed */
+   };
+
+   var includedServiceInitData = {
+     serviceUuid: '5678',
+     isPrimary: true
+   };
+
+   var serviceInitData = {
+     serviceUuid: '1234',
+     includedServices: [includedServiceInitData],
+     characteristics: [characteristicInitData],
+     isPrimary: true
+   };
+   ```
+
+2. Register the service in the local server:
+
+   ```
+   function successCallback() {
+     console.log('Service registered successfully');
+   }
+
+   function errorCallback(error) {
+     console.log('Registering GATT service failed: ' + error.message);
+   }
+
+   var server = tizen.bluetooth.getGATTServer();
+   server.registerService(serviceInitData, successCallback, errorCallback);
+   ```
+   > [!NOTE]
+   > The service will be unregistered from the server when:
+   > - service's `unregister()` method is called,
+   > - server's `unregisterAllServices()` method is called,
+   > - application is reloaded,
+   > - application is closed.
+
+### Unregister services
+GATT services can be unregistered either one at a time or all at once.
+
+To unregister a single service from the local GATT server:
+
+1. Choose the service to be unregistered:
+   ```
+   var server = tizen.bluetooth.getGATTServer();
+   var serviceToBeUnregistered = server.services[3]; 
+   ```
+
+2. Call `unregister()` method:
+   ```
+   function successCallback() {
+     console.log('Service unregistered successfully');
+   }
+
+   function errorCallback(error) {
+     console.log('Unregistering GATT service failed: ' + error.message);
+   }
+
+   serviceToBeUnregistered.unregister(successCallback, errorCallback);
+   ```
+   > [!NOTE]
+   > After `unregister()` is called for the last registered service, the server is stopped.
+   > If new services are then registered, server's `start()` method has to be
+   > called to make them visible to clients.
+
+To unregister all services from the local GATT server at once, call server's
+`unregisterAllServices()` method:
+
+   ```
+   function successCallback() {
+     console.log('All services unregistered successfully');
+   }
+
+   function errorCallback(error) {
+     console.log('Unregistering all GATT services failed: ' + error.message);
+   }
+
+   var server = tizen.bluetooth.getGATTServer();
+   server.unregisterAllServices(successCallback, errorCallback);
+   ```
+   > [!NOTE]
+   > After the calling `unregisterAllServices()`, the server is stopped.
+   > If new services are then registered, server's `start()` method has to be
+   > called to make them visible to clients.
+
+### Send notifications about characteristic's value changes to the clients
+GATT clients connected to the server running on the local device can register for updates of its characteristics' values.
+The server can send two types of such updates - _notifications_ and _indications_.
```

Comment:
In the Tizen docs, we don't recommend to use italic font. So please change " _notifications_ and _indications_" to " `notifications` and `indications`".

The server can send two types of such updates, `notifications` and `indications`.

@surbhig12 Would you please also check the sentence?

Reply:
The sentence is fine.

Reply:
Done